docs/product-user-guide/src/main/asciidoc/business-central-home-con.adoc

```diff
@@ -2,61 +2,162 @@
 = Business Central Home
 
 ifdef::BPMS[]
-The {PRODUCT} *Home* view consists of a main menu with various drop-down options or "perspectives" that you use to author and deploy business process projects, create and manage tasks and dashboards, and integrate plugins.
+The {PRODUCT} Home view provides links to various options, called perspectives, that you use to author and deploy business process projects, create and manage tasks and dashboards, and integrate plugins.
 
 .{PRODUCT} Home
-image::6592.png[BPM Home Screen]
+image::user-guide-homepage.png[BPM Home Screen]
 endif::BPMS[]
 
 ifdef::BRMS[]
-The {PRODUCT} *Home* view consists of a main menu with various drop-down options or "perspectives" that you use to author and deploy business rule projects, create and manage tasks, and integrate plugins.
+The {PRODUCT} Home view provides links to various options, called perspectives, that you use to author and deploy business rule projects, create and manage tasks, and integrate plugins.
 
 .{PRODUCT} Home
-image::3271.png[BRMS Home Screen]
+image::user-guide-homepage.png[BRMS Home Screen]
 endif::BRMS[]
 
-<1> The main menu contains the links to the `Home` page and all available perspectives.
-<2> The perspective menu contains menus for the selected perspective.
-<3> The perspective area contains the perspective tools, such as views and editors.
+In addition, click *Menu* in the top left of the screen to display a list of all of the perspectives. 
 
-These perspectives contain the following options:
+.{PRODUCT} Home Menu
+image::user-guide-homemenu.png[BPM Home Menu]
 
-* *Authoring* group:
-** *Project Authoring* perspective contains:
-*** The *Project Explorer* view with the overview of the repository structure, and information about available resources, such as business process definitions, form definitions, and others.
-*** The editor area on the right of the *Project Explorer* view, where the respective editor appears when a resource is opened.
-*** The *Messages* view with validation messages.
-** *Contributors* perspective enables you to view the number of commits sorted by the organizational unit, repository, author, and other criteria.
-** *Artifact Repository* perspective contains a list of jars which can be added as dependencies. The available operations in this perspective are upload/download artifact and open (view) the `pom.xml` file.
+Click the house icon on the top left of any screen to return to this page.
+
+[NOTE]
+====
+Depending on your role, you may not see all of the perspectives listed. 
```

Comment:
This seems to have a negative connotation. Can we rephrase, something similar to "Perspectives are displayed based on the role of the logged in user."